features/facebookImport/src/facebookImporter.jsx

```diff
@@ -77,10 +77,7 @@ const FacebookImporter = () => {
                     <ReportWrapper />
                 </Switch>
             )}
-            {popUp &&
-                popUp.component({
-                    onClose: closePopUp,
-                })}
+            {popUp && <BaseInfoPopUp />}
```

Comment:
Hmm I'm skeptical here. There are non-info popUps thats should still work with the same popUp logic. 
Either add a new var infoPopUp, if we want to be able to layer them or go back to the old logic I think. It is not in fbImport right now, but I will ensure this will come. In polyExplorer we already have those

Reply:
Hmm, in this case, my proposal is to wait a bit before taking a decision. My next priority is doing the same in polyExplorer (will get started a bit later today) and that will give me more clarity. And I don't want us to merge this PR until the polyExplorer one will be close to merge anyway.

Reply:
Yeah that will help us understand :)